miqa/core/rest/accounts.py

```diff
@@ -88,6 +89,22 @@ def post(self, request):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+class DemoModeLoginView(LoginView):
```

Comment:
It would be a better UX to modify the login template directly to include a new `<form>` with a button that will log the demo user in directly without having to enter any credentials.

Reply:
Added in [83c4e79](https://github.com/OpenImaging/miqa/pull/449/commits/83c4e79006c654ab184c070e51f168e12e3b54ff)